.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project or its docs
+title: ''
+labels: kind/feature
+assignees: ''
+
+---
+
+<!--
+
+Welcome to Apache Airflow!  For a smooth issue process, try to answer the following questions.
+Don't worry if they're not all applicable; just try to include what you can :-)
+
+If you need to include code snippets or logs, please put them in fenced code
+blocks.  If they're super-long, please use the details tag like
+<details><summary>super-long log</summary> lots of stuff </details>
+
+Please delete these comment blocks before submitting the issue.
+
+-->
+
+<!-- What do you want to happen? -->
+
+<!-- Is there currently another issue associated with this? -->
+
+<!-- Does it require a particular kubernetes version? -->
```

Comment:
Should we ask for some motivation for the feature? I would add some required blocks as we do in bug template.

Also why we ask for kube version? :)

Reply:
Because I'm _bad_ at proof reading:

> Oh copy-paste left over from the project I copied these from.

Reply:
Which blocks specifically were you thinking about?

Reply:
For example:
```
** Description **

** Use case / motivation **

** Related to **
```

Without that the request will have any form. And having such blocks will be inline with bug template

Reply:
Probably @michalslowikowski00 can chime in as he has some experience with ticketing :)

Reply:
Take a look at the latest push

Reply:
I am ok with that

Reply:
Yeah, I have some exp. :)
I would add:
** Description **
** Priority/Severity ** 
** Reproduction steps **
** Airflow version **
** Operation System ** 
** Reproduction rate **
** Log **

Reply:
Severity has no bearing on our workflow, so beyond making users feel better I'm not sure what it would do :)

Version, Actual and Expected are there for Bug report already.

OS is very very rarely an issue, so would be noise more often than not.

I've added reproduction rate and logs as comment (not headers) under "Anything else"